odkx-src/build-app.rst

```diff
@@ -831,3 +831,19 @@ To update any app level or table level files, or to modify the database schema (
     .. warning::
 
       Resetting the app server will start a new data set. If you want to keep the old data, you should download it to a separate database.
+
+:th:`To update versions`: 
+
+You need to download the new `app designer <https://github.com/odk-x/app-designer/releases/tag/2.1.7>`_  , and delete the unneeded default files using
```

Comment:
Use `https://github.com/odk-x/app-designer/releases/latest`, this link redirects to the latest tag.